File: 백준/Bronze/2566. 최댓값/최댓값.cc
#include <iostream>
using namespace std;

int main() {
    ios::sync_with_stdio(false); cin.tie(NULL); cout.tie(NULL);
    
    int max_num = -1, max_index[2] = {0, 0};

    for (int i = 1; i <= 9; i++) {
        for (int j = 1; j <= 9; j++) {
            int tmp;
            cin >> tmp;

            if (tmp > max_num) {
                max_num = tmp;
                max_index[0] = i;
                max_index[1] = j;
            }
        }
    }

    cout << max_num << "\n";
    cout << max_index[0] << " " << max_index[1] << "\n";
    
    return 0;
}

File: 백준/Bronze/2566. 최댓값/최댓값.py
import sys

max_num = -1
max_index = [0, 0]

for i in range(9):
    tmp = list(map(int, sys.stdin.readline().split()))
    
    if max(tmp) > max_num:
        max_num = max(tmp)
        max_index[0] = i+1
        max_index[1] = tmp.index(max(tmp)) + 1

sys.stdout.write(f"{max_num}\n")
sys.stdout.write(f"{max_index[0]} {max_index[1]}\n")